src/components/heartrate/Ppg.cpp

```diff
@@ -2,98 +2,242 @@
   SPDX-License-Identifier: LGPL-3.0-or-later
   Original work Copyright (C) 2020 Daniel Thompson
   C++ port Copyright (C) 2021 Jean-François Milants
+  Modified to use frequency space pulse rate metrics and ALS sensor monitoring Copyright (C) 2022 Ceimour
 */
 
 #include "components/heartrate/Ppg.h"
 #include <vector>
 #include <nrf_log.h>
+
 using namespace Pinetime::Controllers;
 
 /** Original implementation from wasp-os : https://github.com/daniel-thompson/wasp-os/blob/master/wasp/ppg.py */
-namespace {
-  int Compare(int8_t* d1, int8_t* d2, size_t count) {
-    int e = 0;
-    for (size_t i = 0; i < count; i++) {
-      auto d = d1[i] - d2[i];
-      e += d * d;
-    }
-    return e;
-  }
+// Refactored for frequency based algorithm - Ceimour
```

Comment:
You don't need to document the history of the changes in the code, git takes care of that :)

Reply:
Cleaned up comments.